datahub-web-react/src/app/search/SearchablePage.tsx

```diff
@@ -140,6 +164,9 @@ export const SearchablePage = ({ onSearch, onAutoComplete, children }: Props) =>
                 authenticatedUserPictureLink={user?.editableProperties?.pictureLink}
                 entityRegistry={entityRegistry}
             />
+            <Helmet>
```

Comment:
Do we need helmet here? 

Ideally the Helmet is defined higher ups in the component hierarchy so that we can change the title for ANYWHERE inside the application

Reply:
Helmet is used in two places APP.tsx and EntityHead. In EntityHead other routes are not wrap and In App.tsx all routes are wrap but dynamicThemeConfig is added in json which is not in context that's the reason I have create new component